process/smartContract/builtInFunctions/changeOwnerAddress.go

```diff
@@ -25,6 +25,10 @@ func NewChangeOwnerAddressFunc(gasCost uint64) *changeOwnerAddress {
 
 // SetNewGasConfig is called whenever gas cost is changed
 func (c *changeOwnerAddress) SetNewGasConfig(gasCost *process.GasCost) {
+	if gasCost == nil {
+		return
```

Comment:
At least a log.Error? So we know something went wrong. Valid on all occurrences

Reply:
added log.error in factory